src/components/ebay-fake-menu-button/examples/separator.marko

```diff
@@ -1,7 +1,7 @@
 <ebay-fake-menu-button text="eBay Menu separator">
     <@item href="#">item 1</@item>
     <@item href="#">item 2</@item>
-    <@separator/>
+    <@item separator/>
```

Comment:
Why are we removing this? Isn't semantically better to have a different component? A separator is not an item that has events and it is completely different with different roles. I feel it is over using the `@item` tag/component

Reply:
It might be semantically better, but Marko has no way of telling which order the attribute tags are in so this required a compiler hook (which is not recommended). Essentially without the compiler hook it looks like this in React:
```jsx
<EbayFakeMenuButton item=[...] separator=[...] />
```